app/components/lazy-mint/LazyMint.tsx

```
@@ -0,0 +1,79 @@
+  const handleSubmitLazyMint = async () => {
+    try {
+      const txnHash = await handleLazyMint({
+        baseURIForTokens: props.baseURIForToken,
+      });
+
+      if (txnHash) {
+        setTxHash(txnHash);
+        props.onSuccess(txnHash);
+
+        toast.success('Lazy Minting', {
+          description: `Transaction hsuccessful`,
```

Biome (1.9.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)